README.md

```diff
@@ -117,6 +117,62 @@ cron.schedule('* * * Jan,Sep Sun', function(){
 });
 ```
 
+## Cron methods
+
+### Schedule
+
+Schedules given task to be executed whenever the cron expression ticks.
+
+Arguments:
+
+- !string expression - Cron expression
+- !Function func - Task to be executed
+- boolean? immediateStart - Whether to start scheduler immediately after create.
```

Comment:
it depends...

if you want it to be cron-like, execute it immediately should be the exception...

either way, won't it make more sense to do it like:

```js
var blah = function () {
  console.log("Working...");
}
cron.schedule("* * * * * *", blah);
blah(); 
```

instead of putting this inside the node-cron api?